Library/Homebrew/Gemfile.lock

```diff
@@ -141,4 +141,4 @@ DEPENDENCIES
   simplecov
 
 BUNDLED WITH
-   1.17.3
+   1.17.2
```

Comment:
don't know why this happened.
restore it ?

Reply:
Yeah revert this chunk.

Reply:
```suggestion
   1.17.3
```

Reply:
The bundler which ships with Portable Ruby is 1.17.2. I'm not sure how 1.17.3 got here (is it different on Catalina system Ruby?) - it's been like that for a couple months judging by warnings spewed every time I update/install gems.

Reply:
@Bo98 thanks that explains it :clinking_glasses: